spec/unit/html_spec.rb

```diff
@@ -75,36 +75,36 @@
-          { :type => "common",   :new_line_number => 7, :original_line_number => 7, :content => "========"},
-          { :type => "common",   :new_line_number => 8, :original_line_number => 8, :content => "- This software is free. You can do with it whatever you want."},
-          { :type => "common",   :new_line_number => 9, :original_line_number => 9, :content => "  I would however kindly ask you to acknowledge the use of this"},
-          { :type => "deletion", :original_line_number => 10, :content => "  package, if you are going use it in your software, which you might"},
-          { :type => "deletion", :original_line_number => 11, :content => "  be going to distribute. I would also like to receive a note if"},
-          { :type => "common",   :new_line_number => 10, :original_line_number => 12, :content => "  you are a satisfied user :-)"},
-          { :type => "addition", :new_line_number => 11, :content => "  Change two"},
-          { :type => "common",   :new_line_number => 12, :original_line_number => 13, :content => nil},
-          { :type => "common",   :new_line_number => 13, :original_line_number => 14, :content => "Acknowledgements:"},
-          { :type => "common",   :new_line_number => 14, :original_line_number => 15, :content => "================="},
-          { :type => "header",   :content => "@@ -20,6 +19,7 @@"},
-          { :type => "common",   :new_line_number => 19, :original_line_number => 20, :content => "==========="},
-          { :type => "common",   :new_line_number => 20, :original_line_number => 21, :content => "- This software is provided ``as is&#39;&#39;. You are using it at your own risk."},
-          { :type => "common",   :new_line_number => 21, :original_line_number => 22, :content => "  I will take no liability in any case."},
-          { :type => "addition", :new_line_number => 22, :content => "  Change three"},
-          { :type => "common",   :new_line_number => 23, :original_line_number => 23, :content => "- This software package uses strong cryptography, so even if it is created,"},
-          { :type => "common",   :new_line_number => 24, :original_line_number => 24, :content => "  maintained and distributed from liberal countries in Europe (where it is"},
-          { :type => "common",   :new_line_number => 25, :original_line_number => 25, :content => "  legal to do this), it falls under certain export/import and/or use"}
+        file: "/etc/postfix/TLS_LICENSE",
+        additions: 3,
+        deletions: 3,
+        lines: [
+          { type: "header",   content: "@@ -1,15 +1,14 @@"},
+          { type: "common",   new_line_number: 1, original_line_number: 1, content: "Author:"},
+          { type: "common",   new_line_number: 2, original_line_number: 2, content: "======="},
+          { type: "common",   new_line_number: 3, original_line_number: 3, content: "- Postfix/TLS support was originally developed by Lutz Jaenicke of"},
```

Comment:
Line is too long. [154/100]<br>Unnecessary spacing detected.<br>Space inside } missing.